cluster-management/setup/cluster-discovery/index.md

```diff
@@ -28,7 +28,7 @@ Boot each one of the machines with identical cloud-config and they should be aut
 #cloud-config
 
 coreos:
-  etcd:
+  etcd2:
```

Comment:
https://coreos.com/docs/running-coreos/cloud-providers/digitalocean/ this page is correct though.